packages/components/src/components/tablist/tablist.component.ts

```diff
@@ -0,0 +1,430 @@
+import { CSSResult, html, nothing } from 'lit';
+import type { PropertyValues } from 'lit';
+import { property, query, queryAssignedElements, state } from 'lit/decorators.js';
+
+import styles from './tablist.styles';
+import { Component } from '../../models';
+import { KEYCODES } from './tablist.constants';
+
+import Tab from '../tab/tab.component';
+import type { ArrowButtonDirection } from './tablist.types';
+import { getFirstTab, getLastTab, getNextTab, getPreviousTab, findTab, getActiveTab } from './tablist.utils';
+
+/**
+ * Tab list organizes tabs into a container.
+ *
+ * Children of the tab list are mdc-tab elements, sent to the default slot.
+ *
+ * The tabs can be navigated using the arrow keys, and selected by clicking, or pressing the Enter and Space keys.
+ *
+ * **Implicit accessibility rules**
+ *
+ * - The element that serves as the container for the set of tabs has role `tablist`.
+ * - Each element that serves as a tab has role `tab` and is contained within the element with role `tablist`.
+ * - The active tab element has the state `aria-selected` set to `true`
+ *   and all other tab elements have it set to `false`.
+ *
+ *
+ * **Accessibility notes for consuming (have to be explicitly set when you consume the component)**
+ *
+ * - Each element that contains the `content panel` for a `tab` has role `tabpanel`.
+ * - If the tab list has a visible label,
+ *   the element with role `tablist` has `aria-labelledby` needs to be a value that refers to the labelling element.
+ *   Otherwise, the `tablist` element needs to have a label provided by `aria-label`.
+ * - Each element with role `tab` has the property `aria-controls`
+ *  that should refer to its associated `tabpanel` element.
+ * - Each element with role `tabpanel` has the property `aria-labelledby` referring to its associated `tab` element.
+ * - If a `tab` element has a popup menu, it needs to have the property `aria-haspopup` set to either `menu` or `true`.
+ *
+ * @tagname mdc-tablist
+ *
+ * @dependency mdc-tab
+ * @dependency mdc-button
+ *
+ * @event change - (React: onChange) This event is dispatched when the tab is selected.
+ *
+ *
+ * Event that fires when user changes the active tab.
+ *     The function sent as the argument will receive the fired event
```

Comment:
```suggestion
 * @event change - (React: onChange) This event is dispatched when the tab is selected.
 * Event that fires when user changes the active tab.
 *     The function sent as the argument will receive the fired event
```

It doesn't matter if you give empty spaces. The description doesn't break into different paragraphs since its described within the attributes table. You can make use of `<br/>` if you really need to.

Reply:
removed the line breaks